public/js/app.js

```diff
@@ -31022,6 +31026,29 @@ $(".track-activity-chart").each(function(index, element) {
     }
   });
 
+  localStorageHasKey = function(key) {
+    return localStorage.getItem(key) !== null;
+  };
+
+  loadCommentFromStorage = function() {
+    return $('#submission_comment').val(localStorage.getItem(location.pathname));
+  };
+
+  recordText = function() {
+    return $('#submission_comment').keyup(function(event) {
+      var nitPickText;
+      nitPickText = $(this).val();
```

Comment:
No need to feel bad! Pickings are always welcome :smile:

I'll make that change real quick like.